float32 yaw
float32 pitch
---
bool state
